x-pack/test/case_api_integration/common/config.ts

```diff
@@ -62,6 +62,35 @@ export function createTestConfig(name: string, options: CreateTestConfigOptions)
       fs.statSync(path.resolve(__dirname, 'fixtures', 'plugins', file)).isDirectory()
     );
 
+    const alertingAllFiles = fs.readdirSync(
```

Comment:
Just noting it here that these changes were so that we can correctly use the alerting test frameworks mock implementation for the connectors.